Adafruit_SSD1306.cpp

```diff
@@ -713,7 +713,7 @@ void Adafruit_SSD1306::drawFastHLineInternal(
     @param  h
             Height of line, in pixels.
     @param  color
-            Line color, one of: BLACK, WHITE or INVERT.
+            Line color, one of: SSD1306_BLACK, _WHITE or _INVERT.
```

Comment:
and here

Reply:
> ... took too long...

me too...
Updated doc strings, ready for you again